swarm/api/http/server.go

```diff
@@ -644,10 +649,15 @@ func (s *Server) logError(format string, v ...interface{}) {
 
 func (s *Server) BadRequest(w http.ResponseWriter, r *Request, reason string) {
 	s.logDebug("bad request %s %s: %s", r.Method, r.uri, reason)
```

Comment:
hmm if we look at the existing current code, sometimes there is a `log.Debug` and sometimes a `log.Error`. Since `ShowError` really just is meant to display an error page to a html client, to me it seems more appropriate to write the log where it is generated, giving the developer the freedom to decide what log level the error requires.